tests/languages/coffeescript/property_feature.test

```diff
@@ -5,11 +5,11 @@ foo_bar :
 
 [
 	["property", "foo"],
-	["punctuation", ":"],
+	["operator", ":"],
 	["property", "foo_bar"],
-	["punctuation", ":"]
+	["operator", ":"]
 ]
 
 ----------------------------------------------------
 
-Checks for object properties.
\ No newline at end of file
+Checks for object properties.

```

Comment:
This is the thing though: it's not an operator here. It's tough to tokenize `:` in a way that distinguishes between its usage in ternary & properties. It's not clear that really solves the issue the original opener had either.

Reply:
I don't know Coffeescript, so what is the `:` here?

Because if it's the same as JS properties (`{ foo: bar }`), then `:` is also an operator in CoffeeScript.

Reply:
Sorry, didn't realize this was coffeescript. I think conceptually, my issue is still the same:

```
const value = { hello: 'world' };
```

The colon here is not an operator, but with this change, will be highlighted as one.